p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateWallet.ExplanationVC.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Created by Giang Long Tran on 01.11.21.
+//
+
+import Foundation
+import UIKit
+
+extension CreateWallet {
+    class ExplanationVC: BaseVC {
+        override var preferredNavigationBarStype: BEViewController.NavigationBarStyle {
+            .hidden
+        }
+        
+        // MARK: - Subviews
+        private lazy var navigationBar: WLNavigationBar = {
+            let navigationBar = WLNavigationBar(forAutoLayout: ())
+            navigationBar.backButton
+                .onTap(self, action: #selector(back))
+            navigationBar.titleLabel.text = L10n.createANewWallet
+            return navigationBar
+        }()
+        
+        private lazy var createWalletButton: UIView = {
```

Comment:
There is a new API for creating this type of buttons in `develop`: 
For button with recommended action, use WLStepButton.main
For button with not-recommended action, use WLStepButton.sub
See "CreateOrRestoreWallet.ViewController for example"